### PR TITLE
Completed Task6

### PR DESCRIPTION
**Pull request by yshurhay**

### CODE REVIEW

**Comment by smiakchilo** [comment]
This is a frontend task. You should have asked for a review from a frontent mentor
BTW: Commit messages are virtually missing. A commit message **must** contain comprehensive info on what exactly has been done in this or that commit. Students have been warned on that multiple times

File: task6/posts.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            case "author":
+                return typeof post.author === "string" && post.author.length !== 0;
+            case "hashTags":
+                return post.hashTags && post.hashTags.every(tag => typeof tag === "string");
+            case "likes":
+                return post.hashTags && post.likes.every(tag => typeof tag === "string");
+            case "photoLink":
+                return typeof post.photoLink === "string";
+            default:
+                return false;
```
Comment (MaksimViarenich) [inline task6/posts.js:118]: Привет Паланевичу

**Review by MaksimViarenich — CHANGES_REQUESTED** [review]

**Review by MaksimViarenich — COMMENTED** [review]
Задание будет принято, если оно будет выполнено Вами

File: UI/style.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
+body {
```
Comment (MaksimViarenich) [inline UI/style.css:1]: Code style

File: task4/OpenTriviaAPI/Open Trivia API.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (MaksimViarenich) [inline task4/OpenTriviaAPI/Open Trivia API.postman_collection.json:1]: In this PR I shouldn't see BE code, it's UI task only

File: task6/posts.js [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+        resultPosts.sort(function (post1, post2) {
+            if(post1.createdAt < post2.createdAt) {
+                return 1;
+            }
+            else if(post2.createdAt < post1.createdAt) {
+                return -1;
+            }
+            else {
+                return 0;
+            }
```
Comment (MaksimViarenich) [inline task6/posts.js:86]: `switch...case`

**Review by MaksimViarenich — APPROVED** [review]
Все еще вижу отсылки к Паланевичу, но защитаю

**Pull request merged**